website/docs/docs/deploy/job-scheduler.md

```diff
@@ -67,32 +91,82 @@ Jobs consume a lot of memory in the following situations:
 Refer to [dbt Cloud architecture](/docs/cloud/about-cloud/architecture) for an architecture diagram and to learn how the data flows.
 
 
-<!-- leaving space for CI job treatment, runtime, warm up pods updates
+### Pod memory 
 
--->
 
-## Run cancelation for over-scheduled jobs
+## Run cancellation for over-scheduled jobs
```

Comment:
copying what Merriam Webster has for the spelling https://www.merriam-webster.com/dictionary/cancellation , which i typically use as a "source of truth" for American English.